docs_outdoornav_user_manual/api/api_examples/mission_from_file.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: Mission from YAML File
+sidebar_label: Mission from YAML File
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+## The Code
+
+``` python
+#! /usr/bin/env python3
+
+from cpr_outdoornav_api_examples_lib.ros_node import RosNode
+from cpr_outdoornav_api_examples_lib.yaml_config import YamlConfig

```

Comment:
At present, cpr_outdoornav_api_examples are only available as private files to partnerbot users. Will non-partnerbot users be confused by this?

Reply:
Likely yes, however I think it might be useful to start getting this information into the documentation. I would just put in a warning on each example and the main API examples page that this feature will only be available to the general public in version 0.10.0. Customers would still be able to follow the examples to write up their files but won't be able to use them yet.